eng/Versions.props

```diff
@@ -66,5 +67,25 @@
     <MicrosoftEntityFrameworkCoreSqlServerPackageVersion>8.0.8</MicrosoftEntityFrameworkCoreSqlServerPackageVersion>
     <MicrosoftEntityFrameworkCoreToolsPackageVersion>8.0.8</MicrosoftEntityFrameworkCoreToolsPackageVersion>
     <MicrosoftNETRuntimeWorkloadTestingInternalVersion>9.0.0-preview.5.24272.3</MicrosoftNETRuntimeWorkloadTestingInternalVersion>
+
+    <NpgsqlEntityFrameworkCorePostgreSQLPackageVersion>8.0.4</NpgsqlEntityFrameworkCorePostgreSQLPackageVersion>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net9.0'">
```

Comment:
I don't understand why we need to do this split. Microsoft.EntityFrameworkCore 9.0-rc1 targets net8.0. Why do we need to keep referencing the version 8 packages when targeting `net8.0`? Why can't we just continue to only target `net8.0` and depend on EF v9.0?

Reply:
I talked about it with @joperezr before starting. efcore 9.0 brings extensions dependencies on 9.0 which we would have to bump (logging, ...), and customers would then need these too, even when targeting 8.0. If we didn't do that users who would like to stay on 9.0 for support reasons would not be able to.

Reply:
I don't like this situation we are putting people (especially library authors) into.

You can reference the 9.0 extension libraries on net8.0. If we cared about this, we should make EF multi-target and not pull in the 9.0 extensions when on net8.0.

We shouldn't have some teams caring about this, and some teams not.

cc @ericstj @SamMonoRT

Reply:
Lots of people care about this because it'll mean going from using extensions from the shared framework, to bin deploying all of extensions for .NET 8 apps.

Reply:
Should we fix EF then? Why does their version `9.0` packages support `net8.0` but lift of MS.Ext references to version `9.0`?

Reply:
Im surprised it doesn't

Reply:
Would this work? https://github.com/dotnet/efcore/pull/34769

Wouldn't it cause downgrade warnings when used in conjunction with other 9.x packages?

Reply:
Can we bump these to the rc.2 versions? Does that line up with the respective planned release dates?